### PR TITLE
Added support for workspace option/buttons

### PR DESCRIPTION
**Pull request by stevenpkg**

### CODE REVIEW

**Comment by stevenpkg** [comment]
@Oliver-I can you follow this pull request as I will be on holiday from Friday

**Comment by stevenpkg** [comment]
car-workspace.json adds

- Action Dialog structures (Turn off, Turn on, Turn up and Turn down) have been streamlined.
- Options (Buttons) added to Action Dialog structures when requesting information.
- Options (Lists) added to Amenity structure.
- Intents stored in Dialog Folder structures.
- Actions renamed to ui_actions.

File: .ibm-project [hunk 1]
```diff
@@ -0,0 +1 @@
+{"build-release-ready":false,"build-debug-ready":false,"devops":false,"deploy-image":"","chart-path":"","ibm-cluster":"","motd-access":"Mon, 30 Jul 2018 10:38:08 AEST","deploy-target":""}
```
Comment (germanattanasio) [inline .ibm-project]: can you delete this file and add it to `.gitignore`

**Comment by germanattanasio** [comment]
@stevenpkg can you deploy this somewhere so that we can test it before merging the code?

**Comment by germanattanasio** [comment]
if you already did it then we can merge this code

**Review by germanattanasio — APPROVED** [review]

**Comment by stevenpkg** [comment]
The test car-dashboard with options/buttons is here:

https://car-dashboard-options.mybluemix.net/

type 'turn on'

**Comment by germanattanasio** [comment]
@stevenpkg can you update this?

**Comment by Oliver-I** [comment]
@stevenpkg Does this mean the Live version is also updated?

**Comment by stevenpkg** [comment]
YesSteve Green (PhD)Watson System DeveloperPhone: +61(0) 416 179 737E-mail: stevenpkg@au1.ibm.com



   Oliver Ivanoski --- Re: [watson-developer-cloud/car-dashboard] Added support for workspace option/buttons (#95) --- 
    From:"Oliver Ivanoski" <notifications@github.com>To:"watson-developer-cloud/car-dashboard" <car-dashboard@noreply.github.com>Cc:"Steve Green" <stevenpkg@au1.ibm.com>, "Mention" <mention@noreply.github.com>Date:Fri., 3 Aug. 2018 8:02 amSubject:Re: [watson-developer-cloud/car-dashboard] Added support for workspace option/buttons (#95)
  
  @stevenpkg Does this mean the Live version is also updated?

**Pull request merged**